### PR TITLE
Remove Ben from about page

### LINKED ISSUE #1879: Remove Ben from about page
RIP Ben. (not for real, but he has moved on from Sentry/Syntax. It feels like *too soon* but gotta do it).

Ben Vinegar is on /about, and is no longer on the team and should be removed.

### PR DESCRIPTION
**Pull request by stolinski**
Fixes #1879

Removes Ben sadly.
Done by robots.

---

For more details, open the [Copilot Workspace session](https://copilot-workspace.githubnext.com/syntaxfm/website/issues/1879?shareId=780423bf-fe0c-443b-8ed9-59cbf362e166).

### CODE REVIEW

**Comment by wesbos** [comment]
Gotta remove the image too

File: src/routes/(site)/about/+page.svelte [hunk 1]
```diff
@@ -4,7 +4,6 @@
 	import { fly } from 'svelte/transition';
 	import emo from '$assets/emo.jpg';
 	import bboy from '$assets/bboy.jpg';
-	import benjamin from '$assets/benjamin.jpg';
```
Comment (wesbos) [inline src/routes/(site)/about/+page.svelte:7]: gotta remove this image from the codebase too

Comment (stolinski) [reply src/routes/(site)/about/+page.svelte:7]: dope thanks for this. going to give workspace another run and will see if we can use your review to delete.

**Comment by wesbos** [comment]
PR'd here https://github.com/syntaxfm/website/pull/1897

**Comment by wesbos** [comment]
https://github.com/syntaxfm/website/pull/1897

**Pull request closed without merging**